docs/develop/dotnet/redis-om-dotnet/add-and-retrieve-objects/add-and-retrieve-objects.md

```diff
@@ -0,0 +1,59 @@
+---
+id: add-and-retrieve-objects
+title: Add and Retrieve Objects
+sidebar_label: Add and Retrieve Objects
+slug: /develop/dotnet/redis-om-dotnet/add-and-retrieve-objects
+---
+
+The Redis OM library supports declarative storage and retrieval of objects out of Redis. Without the RediSearch and RedisJson modules, this is limited to using hashes, and id lookups of objects in Redis. You will still use the `Document` Attribute to decorate a class you'd like to store in Redis. From there, all you need to do is either call `Insert` or `InsertAsync` on the `RedisCollection` or `Set` or `SetAsync` on the RedisConnection, passing in the object you want to set in Redis. You can then retrieve those objects with `Get<T>` or `GetAsync<T>` with the `RedisConnection` or with `FindById` or `FindByIdAsync` in the RedisCollection.
```

Comment:
It might be clearer if you establish that there are collection classes, with operations that take decorated models/classes.